style.css

```diff
@@ -42,8 +42,86 @@ ul {
   gap: 20px;
 }
 
+.nav-menu {
+  display: flex;
+  position: fixed;
+  left: -100%;
+  top: -2px;
+  height: 100vh;
+  gap: 0;
+  width: 100%;
+  text-align: start;
+  flex-direction: column;
+  transition: 0.3s;
+  background-color: rgb(248, 250, 249);
+  padding-left: 40px;
+  padding-top: -10%;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add the missing cancel button to the mobile menu to match the Figma design.
- You could add a `z-index: 1` to make the `.nav-menu` covers other element and so the height takes up the whole device height.

![x](https://user-images.githubusercontent.com/59397159/174705633-4212e259-4143-4b0c-9b08-118b804e9cea.png)